common/config/.golangci-format.yml

```diff
@@ -29,6 +29,7 @@ run:
   skip-files:
   - ".*\\.pb\\.go"
   - ".*\\.gen\\.go"
+  - pilot/pkg/networking/grpcgen/grpcgen_test.go
```

Comment:
why do we need this?

Reply:
This is to skip import order check. Otherwise we will have to set those env vars in the build file

Reply:
oh, thats annoying. Ok, SGTM. I have a bug with GRPC to make this reasonable to configure